zproject/urls.py

```diff
@@ -790,6 +791,10 @@
     path("scim/v2/", include("django_scim.urls", namespace="scim")),
 ]
 
+# Front-end Sentry requests tunnel through the server, if enabled
+if settings.SENTRY_FRONTEND_DSN:
+    urls += [path("sentry-tunnel", sentry_tunnel)]
```

Comment:
We usually use `_` as a separator in URLs we control. Arguably something like `/error_reports` might be more appropriate -- the fact that it's a tunnel is maybe not important (I guess `/json/report_error` was the blueslip frontend endpoint, but that's a bit less of a REST style naming), but I'm fine with `/sentry_tunnel` too.

Reply:
It's not just used for error reports -- session traces go there as well.  I've repushed with `/error_tracing` as the name.